Src/Newtonsoft.Json.Schema/JSchemaReaderSettings.cs

```diff
@@ -26,6 +26,12 @@ public class JSchemaReaderSettings
         public JSchemaResolver Resolver { get; set; }
 
         /// <summary>
+        /// Sets a flag requesting the <see cref="JSchemaResolver"/> to dereference schema references.
+        /// </summary>
+        /// <remarks>Default is <c>true</c></remarks>
+        public bool ResolveDeferedSchemas { get; set; } = true;
```

Comment:
"DeferedSchemas" is more of an internal idea rather than something the user needs to worry about. What about `ResolveSchemaReferences`?